README.md

```diff
@@ -174,7 +175,6 @@ Binaries will be published once the codebase reaches a stable condition.
 | [MarkDzulko](https://github.com/MarkDzulko) | 2 |
 | [gam-phon](https://github.com/gam-phon) | 2 |
 | [cornelk](https://github.com/cornelk) | 2 |
-| [if1live](https://github.com/if1live) | 2 |
```

Comment:
User if1live can be added to the hardcoded contributors list here: https://github.com/samuael/gocryptotrader/blob/okx/cmd/documentation/documentation.go#L285